vnext/ReactUWP/ReactUWP.vcxproj.filters

```diff
@@ -295,6 +295,7 @@
     <ClCompile Include="Threading\BatchingUIMessageQueueThread.cpp">
       <Filter>Threading</Filter>
     </ClCompile>
+    <ClCompile Include="Modules\StatusBarModule.cpp" />
```

Comment:
><ClCompile Include="Modules\StatusBarModule.cpp" /> [](start = 4, length = 51)

This change was probably unintentional?

Reply:
I think these lines were automatically added by visual studio, since the corresponding files are in the ReactUWP.vcxproj but were not in this filter file. I think we should keep them, and have moved them under the Modules filter for clarity,

---
In reply to: [330318564](https://github.com/microsoft/react-native-windows/pull/3296#discussion_r330318564) [](ancestors = 330318564)